packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -12,20 +51,108 @@ export default {
     name: 'ImageTile',
     components: {},
     props: {
+        href: {
+            type: String,
+            default: 'https://www.google.com'
+        },
+        tileId: {
+            type: String,
+            default: 'Chicken'
+        },
+        isSelected: {
+            type: Boolean,
+            default: false
+        },
+        isLink: {
+            type: Boolean,
+            default: false
+        },
+        displayText: {
+            type: String,
+            default: 'Chicken'
+        },
+        imgSrc: {
+            type: String,
+            default: 'https://via.placeholder.com/150'
+        }
+    },
+    data () {
+        return {
+            isFilterSelected: false
+        };
+    },
+    watch: {
+        isSelected (newValue) {
+            if (newValue !== this.isFilterSelected) {
+                this.isFilterSelected = newValue;
+            }
+        }
+    },
+    mounted () {
+        this.isFilterSelected = this.isSelected;
+    },
+    methods: {
+        /**
+         * Toggles the isFilterSelected data property
+         * when the checkbox is toggled.
+         *
+         */
+        toggleFilter () {
+            this.isFilterSelected = !this.isFilterSelected;
+
+            this.$nextTick(() => {
+                this.$emit('toggle', this.filterEmitObject(this.tileId));
+            });
+        },
+        /**
+         * Creates the object that is $emitted
+         * when the checkbox is toggled
+         *
+         * @param {tileId} string
+         */
+        filterEmitObject (tileId) {
+            return {
+                tileId
+            };
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
 
 .c-imageTile {
-    display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    font-family: $font-family-base;
     @include font-size(heading-m);
+
+    font-family: $font-family-base;
+    position: relative;
+}
+
+.c-imageTile-link {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+}
+
+.c-imageTile-link--toggle {
+  display: block;
+  position: static;
+  pointer-events: none;
+}
+
+.c-imageTile-label {
+    display: flex;
+    flex-flow: column wrap;
+}
+
+.c-imageTile-label--selected {
+  border: 1px solid green;
+}
+
```

Comment:
^ I'm using this as a visual guide whilst developing and before the components actual styles are added. This will be removed but is needed for now